### ISSUE TITLE
`docker-compose/architect.yml` removed when running dev in detached mode

### ISSUE THREAD

**Issue by cedric-neighborschools** [architect-team/architect-cli#772]
When running `architect dev -d`, monitor the `~/.config/architect/docker-compose/` directory. The `architect.yml` file will get created while Architect is booting up, but as soon as it exits into detached mode the `architect.yml` gets removed, and then commands can't be run using `architect exec`:

```sh
$ architect dev -d

$ ls -al ~/.config/architect/docker-compose/
total 16
drwxrwxr-x 2 cedric cedric 4096 Dec  5 07:20 .
drwxrwxr-x 3 cedric cedric 4096 Nov 29 08:45 ..
-rw-rw-r-- 1 cedric cedric  306 Nov 17 11:26 register.forum.latest.yml
-rw-rw-r-- 1 cedric cedric  291 Oct  7 14:18 register.guidepost.latest.yml

$ architect exec --environment architect component.services.app -- ls

ENOENT: no such file or directory, open '/home/cedric/.config/architect/docker-compose/architect.yml'
    Error: ENOENT: no such file or directory, open '/home/cedric/.config/architect/docker-compose/architect.yml'
    Code: ENOENT
```

**Comment by ryan-cahill** [comment]
@cedric-neighborschools this was fixed with https://github.com/architect-team/architect-cli/pull/756 and released with CLI version 1.29.0 last week. Can you please update your CLI and let us know if this is still an issue?

**Comment by cedric-neighborschools** [comment]
Woops, didn't catch that. It works after the update. Thanks!
